### PR TITLE
epub_reduced size

### PR DESCRIPTION
**Pull request by Stoneguard001**
A simple change to create a reduced size color epub for readers that have color but a size limit. The resulting file is ~30MB

### CODE REVIEW

**Review by cedounet — CHANGES_REQUESTED** [review]
Looking good.

i don’t like cl prefix as it doesn’t automatically means anything to me.  How about low_res ?

watch your trailing spaces and extra empty lines (or missing new line at end of files) while doing the changes and we should be in a good place.

We would need to discuss if we want a high res ebook and a low res one or not.  Not a ebook user myself so I won’t comment.

File: .gitignore [hunk 1]
```diff
@@ -41,3 +41,4 @@ book/epub/
 book/book-epub/
 book/bw-book-epub/*
 book/release/*
+book/low-res-book-epub/*
```
Comment (Stoneguard001) [inline .gitignore:44]: not sure why the newline is not showing here.

**Comment by Stoneguard001** [comment]
I think I have the changes you want, though I'm not sure why the .gitignore is not showing that I added a newline on the last line. I'm so used to prettier handling all my formatting, I forgot about it.

I agree there should probably be a discussion about whether there needs to be two color epubs; I personally don't see the benefit of the full-res images in the epub. Not sure where that discussion should happen. Maybe on your discord? (I'm not yet a subscriber, so don't have access to it just yet).

**Comment by cedounet** [comment]
I will test it then merge it for now...  The ebook size has been a recurring topic but I thought I nailed it with B&W version.

I know that @hendricius has plans to replace some pictures, and IMHO some pictures today are too large even for the pdf.  I just converted one png to jpg and it went from 1.3 MBytes to 165K.

**Comment by cedounet** [comment]
@Stoneguard001 I just realized png are not systematically smaller in your lowres version... nothing dramatic though
`
370K Dec 27 14:52 book-epub/OEBPS/baking/baking-experiment-temperatures.png
 18K Dec 27 14:52 book-epub/OEBPS/cover/CC-BY-SA.png
287K Dec 27 14:52 book-epub/OEBPS/images/external/wheat-kernel-overview.png
100K Dec 27 14:52 book-epub/OEBPS/troubleshooting/crumb-structures/crumb-structures-book.png
 33K Dec 27 14:52 book-epub/OEBPS/wheat-sourdough/dough-strength-sourdough-yeast.png
 48K Dec 27 14:52 book-epub/OEBPS/wheat-sourdough/dough-strength-sourdough.png
 69K Dec 27 14:52 book-epub/OEBPS/wheat-sourdough/dough-surface-touchpoints.png

> ll low-res-book-epub/OEBPS/**/*.png    
415K Dec 27 14:53 low-res-book-epub/OEBPS/baking/baking-experiment-temperatures.png
11K Dec 27 14:53 low-res-book-epub/OEBPS/cover/CC-BY-SA.png
327K Dec 27 14:53 low-res-book-epub/OEBPS/images/external/wheat-kernel-overview.png
131K Dec 27 14:53 low-res-book-epub/OEBPS/troubleshooting/crumb-structures/crumb-structures-book.png
52K Dec 27 14:53 low-res-book-epub/OEBPS/wheat-sourdough/dough-strength-sourdough-yeast.png
74K Dec 27 14:53 low-res-book-epub/OEBPS/wheat-sourdough/dough-strength-sourdough.png
116K Dec 27 14:53 low-res-book-epub/OEBPS/wheat-sourdough/dough-surface-touchpoints.png
`

**Comment by Stoneguard001** [comment]
Yes, the dramatic changes were the larger jpg files.  I am not sure what resolutions are best for print, but assumed that was the primary focus of the larger images. I think as long as the final size of the epub is under 50MB it's good.

**Comment by hendricius** [comment]
Great PR. Thank you!

**Pull request merged**